catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/pojo/CommonTemplate.java

```diff
@@ -53,6 +62,20 @@ public CommonTemplate(Metacard metacard) {
     this.owner = safeGet(metacard, Core.METACARD_OWNER, String.class);
   }
 
+  public CommonTemplate(
+      Metacard metacard, Map<String, List<Serializable>> securityAttributes) {
+    this.id = safeGet(metacard, Core.ID, String.class);
+    this.title = safeGet(metacard, Core.TITLE, String.class);
+    this.description = safeGet(metacard, Core.DESCRIPTION, String.class);
+
+    this.created = safeGet(metacard, Core.CREATED, Date.class);
+    this.modified = safeGet(metacard, Core.MODIFIED, Date.class);
+    this.owner = safeGet(metacard, Core.METACARD_OWNER, String.class);
+
+    this.accessIndividuals = securityAttributes.get(Security.ACCESS_INDIVIDUALS);
```

Comment:
❓ Do we need to worry about this value being `null`? i.e. `getOrDefault(..., new ArrayList<>())`?